src/libs/KeyboardShortcut/index.js

```diff
@@ -74,19 +72,20 @@ const KeyboardShortcut = {
      * @param {Boolean} captureOnInputs Should we capture the event on inputs too?
      */
     subscribe(key, callback, modifiers = 'shift', captureOnInputs = false) {
-        const keyCode = key.charCodeAt(0);
-        if (events[keyCode] === undefined) {
-            events[keyCode] = [];
+        // Enable support for special keys like Escape
+
```

Comment:
Extra line here can be removed.

Reply:
Unfortunately I couldn't reproduce issue with outline at my end... can you state platform and OS you used?